r-package/survlearners/R/surv_rl_grf.R

```diff
@@ -86,15 +86,14 @@ surv_rl_grf <- function(X, Y, W, D,
   args.grf.nuisance[names(new.args.grf.nuisance)] <- new.args.grf.nuisance
 
```

Comment:
Just above note that `compute.oob.predictions = FALSE`  for Erik's S-learner workaround; however this would give the wrong result here.